res/database/Default/database_ads.xml

```diff
@@ -617,7 +618,7 @@ But only with Relocation Rudolph!</en>
 			</title>
 			<description>
 				<de>Der Schorotti-Mohr bietet traumhafte Schokolade. Zugreifen und dahinschmelzen.</de>
-				<en>The Schorotti-Maure offers fantastic chocolate. Take one and melt away.</en>
+				<en>The Schorotti-Maure-Mallow features fantastic chocolate. Take one and melt away.</en>
```

Comment:
https://en.wikipedia.org/wiki/Chocolate-coated_marshmallow_treats

Reply:
But the "Sarotti-Mohr" really is the mascot of the company, so "Schorotti-Maure" seems to be fine.

Reply:
Don't forget to remove the add after year x...they removed the maure too (2020).

Re "Mallow":
Dunno how much the association to the "Mohrenköpfe" might become more of a problem then ("maure-mallow").
So maybe simply keep it "Schorotti-Maure"

Reply:
The thing is that the "Maure" has never existed in this context outside of the German speaking region in the first place. The linked wikipedia (which is the direct English version of "Schokokuss") has a short section about this peculiarity with the "Mohr" in German.

But because "Chocolate-coated marshmallow" or the like is too convoluted, I tried to come up with something that keeps the original "Maure" but puts it in a way that it could read like the deliberate name of a product of an imaginary candy company "Schorotti", hence the "Maure-Mallow" (instead of Marshmallow).

And good point, we could make a new version of this ad for after 2020 and make fun of why they kept such a problematic product name for such a long time.

Reply:
keeping the Maure-Mallow but without the first hyphen. Adding TODO for version of the ad after 2020.